docs/content/preview/yugabyte-voyager/migrate/live-fall-back.md

```diff
@@ -204,7 +204,7 @@ Create a new database user, and assign the necessary user permissions.
 
     CREATE TABLE ybvoyager_metadata.ybvoyager_imported_event_count_by_table (
                   migration_uuid VARCHAR2(36),
-                  table_name VARCHAR2(250),
+                  table_name VARCHAR2(1000),
```

Comment:
Hey @sudhanshu456 , after further discussion, we decided to make this 4000 instead of 1000. Can you pls make this change?

Reply:
```suggestion
                  table_name VARCHAR2(4000),
```